src/java.base/share/classes/java/net/URL.java

```diff
@@ -1589,7 +1589,7 @@ static boolean isOverrideable(String protocol) {
     /**
      * A table of protocol handlers.
      */
-    static Hashtable<String,URLStreamHandler> handlers = new Hashtable<>();
+    static final Hashtable<String,URLStreamHandler> handlers = new Hashtable<>();
```

Comment:
Can you check if this can also be changed to be private?

Reply:
sure. Done